raster/r.neighbors/main.c

```diff
@@ -231,6 +232,12 @@ int main(int argc, char *argv[])
     parm.gauss->required = NO;
     parm.gauss->description = _("Sigma (in cells) for Gaussian filter");
 
+    parm.exp = G_define_option();
+    parm.exp->key = "exponent";
+    parm.exp->type = TYPE_DOUBLE;
+    parm.exp->required = NO;
+    parm.exp->description = _("Factor for an exponential kernel");
```

Comment:
Not sure I would understand what is meant by 'factor', here. Why not simple call it 'Exponent for an exponential kernel' ?